server/models.py

```diff
@@ -6,7 +6,12 @@
 from django.contrib.auth.models import User as DJUser
 
 import sys, datetime, logging
-
+#Vending Machine Stock
+class Venditem(models.Model):
+  name = models.TextField(default = '')
+  item = models.PositiveIntegerField()
+  price = models.TextField(default = '')
```

Comment:
Price is a numeric value, text is not the right way to store it!

I was about to say the only correct way of storing monetary values is an integer field (containing pence values) however actually Django has a fixed point [DecimalField](https://docs.djangoproject.com/en/2.1/ref/models/fields/#django.db.models.DecimalField) type which might do the trick here. A word of caution though: don't handle currency with floating point math functionality - rounding errors can creep in and make a mess of things. Again, it's best to use integers here.

Given that last point, I'm actually leaning towards saying an integer field is the best choice here to avoid the temptation to process the values as floating points.

Reply:
I've set price to be stored as an integer (containing pence values) `balance = models.PositiveIntegerField(default=0)`